src/common/components/__tests__/ExternalLink.test.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ExternalLink from '../ExternalLink';
+
+describe('ExternalLink', () => {
+  it('renders with only href and children', () => {
+    const wrapper = shallow(
+      <ExternalLink href="//example.com">example</ExternalLink>
+    );
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('renders with only href, children and extra props', () => {
+    const wrapper = shallow(
+      <ExternalLink href="//example.com" className="test">
+        example
+      </ExternalLink>
```

Comment:
what about adding a test with _blank for example?

Reply:
`_blank` is already by default set to the `anchor` since this is external link. But do you mean testing this like a failure or wrong case?

Reply:
Maybe I'm wrong but I don't see any case to check deeper in the ExternalLink, if the link has all the attributes etc